plans/example/drop.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
We should either come up with a simple plan (`splitbrain` seems to be it), or just include this snippet in the docs website. I don't see a reason to have it in `plans/example` as it is not really a plan and doesn't do anything, but rather just showcases functionality that you could use.